tensorflow/python/ops/init_ops.py

```diff
@@ -419,6 +421,8 @@ class VarianceScaling(Initializer):
   With `distribution="uniform"`, samples are drawn from a uniform distribution
   within [-limit, limit], with `limit = sqrt(3 * scale / n)`.
 
+  `distribution="normal"` is a deprecated alias for "truncated_normal".
```

Comment:
If you want, you can use @deprecated_arg_values to formally encode this (with a warning). But I think what you did here is fine.